apps/meteor/ee/server/local-services/voip-freeswitch/service.ts

```diff
@@ -1,27 +1,53 @@
 import { type IVoipFreeSwitchService, ServiceClassInternal } from '@rocket.chat/core-services';
-import type { FreeSwitchExtension, ISetting, SettingValue } from '@rocket.chat/core-typings';
-import { getDomain, getUserPassword, getExtensionList, getExtensionDetails } from '@rocket.chat/freeswitch';
+import type {
+	DeepPartial,
+	IFreeSwitchEventCall,
+	IFreeSwitchEventCaller,
+	IFreeSwitchEvent,
+	FreeSwitchExtension,
+	IFreeSwitchCall,
+	IFreeSwitchCallEventType,
+	IFreeSwitchCallEvent,
+	AtLeast,
+} from '@rocket.chat/core-typings';
+import { isKnownFreeSwitchEventType } from '@rocket.chat/core-typings';
+import { getDomain, getUserPassword, getExtensionList, getExtensionDetails, listenToEvents } from '@rocket.chat/freeswitch';
+import type { InsertionModel } from '@rocket.chat/model-typings';
+import { FreeSwitchCall, FreeSwitchEvent, Users } from '@rocket.chat/models';
+import { objectMap, wrapExceptions } from '@rocket.chat/tools';
+import type { WithoutId } from 'mongodb';
+
+import { settings } from '../../../../app/settings/server';
 
 export class VoipFreeSwitchService extends ServiceClassInternal implements IVoipFreeSwitchService {
 	protected name = 'voip-freeswitch';
 
-	constructor(private getSetting: <T extends SettingValue = SettingValue>(id: ISetting['_id']) => T) {
```

Comment:
Any reason for stopping injecting the setting-getter here? 🤔 
Also, if this is not necessary for the feature itself, we could keep it in another PR.